### PR TITLE
Feature hit judgements

### PR DESCRIPTION
**Pull request by CaptSiro**
Inside the function `createChartConfig` you can fiddle with colours of the graph.

Condition `index.js:241` probably needs a bit more tweaking but generally it should work

### CODE REVIEW

File: js/GraphFill.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+
+
+    /**
+     * @param {Color} fill
+     * @param border
+     */
+    constructor(fill, border = undefined) {
+        this.#fill = fill;
+        this.#border = border ?? fill.clone().setAlpha(1);
```
Comment (AutumnVN) [inline js/GraphFill.js]: dont use ?? it breaks ingame overlay
```suggestion
        this.#border = border !== undefined ? border : fill.clone().setAlpha(1);
```

Comment (vifiiy) [reply js/GraphFill.js]: true

**Review by vifiiy — CHANGES_REQUESTED** [review]
@CaptSiro Could you resolve the merge conflicts by merging with main and resolving them?

File: js/GraphFill.js (hunk 1 above)
Comment (CaptSiro) [reply js/GraphFill.js]: I wonder how because `a ?? b` is just syntax sugar for `a === undefined ? b : a`

Comment (vifiiy) [reply js/GraphFill.js]: tosu's typescript doesnt really like it

Comment (brw) [reply js/GraphFill.js]: Nullish coalescing operator was added in ES2020. Perhaps whatever JS engine tosu uses for its ingame overlay is just an older version.

**Review by vifiiy — APPROVED** [review]
lgtm!

**Pull request merged**